.github/workflows/ci.yml

```diff
@@ -47,61 +46,77 @@ jobs:
         run: |
           swift -version
           swift package --version
-      # FIXME: suppress warning for Nimble
       - name: Build and Test
-        run: swift test -Xswiftc -suppress-warnings
+        run: swift test
 
-  combine:
-    runs-on: macOS-latest
-    env:
-      CX_COMBINE_IMPLEMENTATION: Combine
-      CX_CONTINUOUS_INTEGRATION: "YES"
+  swift5_0:
+    strategy:
+      matrix:
+        swift_version: ['5.1'] # FIXME: swift 5.0
+    runs-on: ubuntu-latest
+    container:
+      image: swift:${{ matrix.swift_version }}
     steps:
       - uses: actions/checkout@v1
       - name: Swift Version
         run: |
-          sw_vers -productVersion
           swift -version
           swift package --version
       - name: Build and Test
-        # FIXME: suppress warning for Nimble
-        run: swift test -Xswiftc -suppress-warnings
+        # no tests available, just build.
+        run: swift build
```

Comment:
I dropped testability for Swift 5.0. Or we got very tricky dependency (Swift 5.0 -> Quick 2.0, Swift 5.2 -> Quick 3.0) and need a bunch of `#if compiler(>=5.2)` in the code base.

Reply:
looks good to me